### DISCUSSION TITLE
Fill out Medley Interlisp History

### DISCUSSION THREAD

**Discussion by blakemcbride** [Interlisp/medley#46] [Implementation history]
As the years pass we lose the history of InterLisp in general and, Medley and maiko in particular.  I request that those who were around during all of this add a file "History.txt" that will include as much of this history as possible.  It is very interesting and important to me and others are likely to feel similarly.

**Comment by masinter** [comment]
The "Medley Interlisp History Timeline" is at
https://docs.google.com/document/d/1QXUMQ0OUttYYzWUvgdVcnkz5Gym_t1MDnMT8XZJe-NI/edit?usp=sharing

The idea was to migrate to replace https://Interlisp.org#history  (from https://GitHub.com/interlisp/interlisp.github.io  
once filled out with more recent events ... want to help?

**Comment by masinter** [comment]
https://github.com/dspinellis/unix-history-make might be a cool way to show Interlisp history from 1970s. I think the tools provided by @dspinellis combined with the snapshots from @larsbrinkhoff  https://github.com/PDP-10/Interlisp-10, PARC backups, Envos and Venue
